Gemfile

```diff
@@ -1,7 +1,17 @@
 source "https://rubygems.org"
 
-gem "github-pages", group: :jekyll_plugins
 gem "jekyll-include-cache"
 gem "webrick"
 gem "html-proofer"
 gem "ffi", "= 1.16.3"
+
+gem "jekyll", "~>4.3.3"
```

Comment:
@sneakers-the-rat i am going to merge this but i have a question.

You may have said this in the pr header but i want to be clear about what this does! 
 Is going this route going to allow us to avoid that github pages restriction of using other jekyll plugins and gems if we can also write our own? I just know that for a while that has also been a constraint for the site.

Reply:
Yep! We can add whatever gems we want here

Reply:
yay!!